kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -103,7 +103,6 @@ def __init__(self, name, init_config, instances):
             'kube_job_owner': {'metric_name': 'job.count', 'allowed_labels': ['namespace', 'owner_name', 'owner_kind']},
             'kube_deployment_status_condition': {
                 'metric_name': 'deployment.count',
-                'allowed_labels': ['namespace', 'condition', 'status'],
```

Comment:
we're also removing `namespace` here?

Reply:
I am not using `count_objects_by_tags` anymore but a self written method `def kube_deployment_count()`